packages/@aws-cdk/custom-resource-handlers/lib/custom-resources-framework/utils/runtime-determiner.ts

```diff
@@ -0,0 +1,109 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import { Runtime, RuntimeFamily } from '../runtime';
+
+/**
+ * A utility class used to determine the latest runtime for a specific runtime family
+ */
+export class RuntimeDeterminer {
```

Comment:
We are *specifying* the latest runtime now, are we not? It's just always NODE_18 or whatever? (Which we *know* is not deprecated?) So why do we still need this?